feature/about/src/test/java/io/github/droidkaigi/confsched2023/AboutScreenTest.kt

```diff
@@ -38,4 +38,14 @@ class AboutScreenTest {
             checkScreenCapture()
         }
     }
+
+    @Test
+    @Category(ScreenshotTests::class)
+    fun checkScrollShot() {
+        aboutScreenRobot {
+            setupAboutScreenContent()
+            scrollAboutScreen()
+            checkAboutContentColumnCapture()
```

Comment:
From the screenshot, it seems to be capturing only a single item, but I was thinking that it would be good to have a screen-level screenshot that includes the behavior when scrolling through the entire screen. At the item level, a preview might probably suffice. My intention was to check the overall behavior of the entire screen, not just an individual item. 🙇